code/modules/mob/living/carbon/human/species_types/flypeople.dm

```diff
@@ -36,3 +42,62 @@
 	if(istype(weapon, /obj/item/melee/flyswatter))
 		return 29 //Flyswatters deal 30x damage to flypeople.
 	return 0
+
+/obj/item/organ/heart/fly
+	name = "???"
+	desc = "You have no idea what the hell this is, or how it manages to keep something alive in any capacity."
+
+/obj/item/organ/heart/fly/Initialize()
+	. = ..()
+	icon_state = pick("brain-x-d", "liver-x", "kidneys-x", "stomach-x", "lungs-x", "random_fly_1", "random_fly_2", "random_fly_3", "random_fly_4", "random_fly_5")
+
+/obj/item/organ/heart/fly/update_icon_state()
+	return //don't set icon thank you
+
+/obj/item/organ/lungs/fly
+	name = "???"
+	desc = "You have no idea what the hell this is, or how it manages to keep something alive in any capacity."
+
+/obj/item/organ/lungs/fly/Initialize()
+	. = ..()
+	icon_state = pick("brain-x-d", "liver-x", "kidneys-x", "stomach-x", "lungs-x", "random_fly_1", "random_fly_2", "random_fly_3", "random_fly_4", "random_fly_5")
+
+/obj/item/organ/liver/fly
+	name = "???"
+	desc = "You have no idea what the hell this is, or how it manages to keep something alive in any capacity."
+	alcohol_tolerance = 0.007 //flies eat vomit, so a lower alcohol tolerance is perfect!
+
+/obj/item/organ/liver/fly/Initialize()
+	. = ..()
+	icon_state = pick("brain-x-d", "liver-x", "kidneys-x", "stomach-x", "lungs-x", "random_fly_1", "random_fly_2", "random_fly_3", "random_fly_4", "random_fly_5")
+
+/obj/item/organ/stomach/fly
+	name = "???"
+	desc = "You have no idea what the hell this is, or how it manages to keep something alive in any capacity."
+
+/obj/item/organ/stomach/fly/Initialize()
+	. = ..()
+	icon_state = pick("brain-x-d", "liver-x", "kidneys-x", "stomach-x", "lungs-x", "random_fly_1", "random_fly_2", "random_fly_3", "random_fly_4", "random_fly_5")
+
+/obj/item/organ/appendix/fly
+	name = "???"
+	desc = "You have no idea what the hell this is, or how it manages to keep something alive in any capacity."
+
+/obj/item/organ/appendix/fly/Initialize()
+	. = ..()
+	icon_state = pick("brain-x-d", "liver-x", "kidneys-x", "stomach-x", "lungs-x", "random_fly_1", "random_fly_2", "random_fly_3", "random_fly_4", "random_fly_5")
+
+/obj/item/organ/appendix/fly/update_icon()
+	return //don't set name or icon thank you
+
+//useless organs we throw in just to fuck with surgeons a bit more
+/obj/item/organ/fly
+	name = "???"
```

Comment:
would be cooler if these were assigned slightly randomized names, you could have an adjective pool of things like "writing," "twitching," "dripping," "soggy," "flaccid," "darkened," "unidentifiable," "crusty," etc then just make the name generated like "[adjective] [adjective] organ"

Reply:
alright, i'll do it. great suggestion